docs/source/conf.py

```diff
@@ -0,0 +1,63 @@
+# Configuration file for the Sphinx documentation builder.
+#
+# This file only contains a selection of the most common options. For a full
+# list see the documentation:
+# https://www.sphinx-doc.org/en/master/usage/configuration.html
+
+# -- Path setup --------------------------------------------------------------
+
+# If extensions (or modules to document with autodoc) are in another directory,
+# add these directories to sys.path here. If the directory is relative to the
+# documentation root, use os.path.abspath to make it absolute, like shown here.
+#
+import os
+import sys
+
+sys.path.insert(0, os.path.abspath(os.path.join("..", "..")))
+
+
+# -- Project information -----------------------------------------------------
+
+project = "git-theta"
+copyright = "2023, r-three"
+author = "r-three"
+
+# The full version, including alpha/beta/rc tags
+release = "0"
```

Comment:
When we actually use these docs I think we should be able to import git_theta here and have the release number be the actually version number.

Reply:
Done.